drm-howto/modeset-atomic.c

```diff
@@ -614,11 +635,6 @@ int main(int argc, char **argv)
 	else
 		card = "/dev/dri/card0";
 
-	if (argc > 3) {
-		preferred_w = atoi(argv[2]);
-		preferred_h = atoi(argv[3]);
-	}
-
```

Comment:
This is not "Add comments to atomic API example" if you go changing the code like this. This kind of changes need their own justification and their own commit.

The commit message should always answer the question "why did you do this change?". If the commit contains changes where the answer does not apply, those changes might need to be in a separate commit.

For example right here, I don't know why you are removing these `atoi()` calls. Are you removing command line options? Are you moving these to some other place in the code (which is hard to see because of all the doc changed)?

Reply:
Argh, I thought I was reading the first commit of the series, not the last one. Sorry.

You should just squash both commits. You make reviewers do double work, when you submit a series where first commit adds something new, and the second commit only changes the new thing. It would be best to just have one commit that adds the whole new thing the way it's meant to finally be when there is no old code to be significantly changed.

For giving credit where it is due, keep the original author intact but explain in the commit message what you changed.